packages/teams-js/src/private/hostEntity/hostEntity.ts

```diff
@@ -30,6 +30,25 @@ export interface TeamsChannelMeetingEntityIds extends TeamsEntityId {
   parentMessageId: string;
 }
 
+/**
+ * @hidden
+ * @internal
+ * @beta
+ * Limited to Microsoft-internal use
+ *
+ * Additional parameters associated with a meeting
+ */
+export interface TeamsExtraMeetingInputs {
```

Comment:
Echoing comments on the host sdk side, it would be better to avoid naming this "Teams" since that is host-specific, "Extra" is implied by this being optional in the function interface, and "Inputs" describes how the interface is used more than what it is. Would something like "MeetingSettings", "MeetingProperties", or something along those lines be appropriate here?